toptal/picasso
# fix: remove spacing on the left from an iconless sidebar menu item

[FX-375](https://toptal-core.atlassian.net/browse/FX-375)

### How to test

1. create SidebarItem without `icon` prop
2. notice there's no more space on the left of it

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| ![before](https://user-images.githubusercontent.com/54671509/64008372-ab9a8a00-cb1e-11e9-805b-2cc6505137e5.png) | ![after](https://user-images.githubusercontent.com/54671509/64008397-b5bc8880-cb1e-11e9-8626-0f5742beab9c.png)|

### Review
